yarn-project/circuit-types/src/notes/note_filter.ts

```diff
@@ -15,4 +15,6 @@ export type NoteFilter = {
   storageSlot?: Fr;
   /** The owner of the note (whose public key was used to encrypt the note). */
   owner?: AztecAddress;
+  /** The status of the note. Defaults to 'activeOnly'. */
+  status?: 'activeOnly' | 'includeNullified'; // TODO: add 'nullifiedOnly'
```

Comment:
When creating a todo that will not be handled as part of the pr, please add a separate issue and link it in the comment, that way we should more easily remember it.